docs/kyma/docs/026-details-testing.md

```diff
@@ -57,25 +56,25 @@ spec:
 
 ## Test execution
 
-All tests created for [core](../../../resources/core/) charts run automatically after starting Kyma.
+All tests created for charts under `/resources/core/` run automatically after starting Kyma.
 If any of the tests fail, the system prints the Pod logs in the terminal, then deletes all the Pods.
 
 >**NOTE:** If you run Kyma locally, by default, the system does not take into account the test's exit code. As a result, the system does not terminate Kyma Docker container, and you can still access it.
 To force a termination in case of failing tests, use `--exit-on-test-fail` flag when executing `run.sh` script.
 
 CI propagates the exit status of tests. If any test fails, the whole CI job fails as well.
 
-Follow the same guidelines to add a test which is not a part of any **core** component. However, for test execution, see **Run a test manually** in this document.
+Follow the same guidelines to add a test which is not a part of any `core` component. However, for test execution, see the **Run a test manually** section in this document.
 
 ### Run a test manually
 
-To run a test manually, use the [testing.sh](../../../installation/scripts/testing.sh) script, which runs all tests defined for **core** releases.
+To run a test manually, use the `testing.sh` script under the `/installation/scripts/` directory, which runs all tests defined for `core` releases.
```

Comment:
script under the `/installation/scripts/` directory -> not a huge fan of "under". How about scripts located in the (...) directory 
?

Reply:
Sure, why not.